packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts

```diff
@@ -274,3 +273,15 @@ function getMockDeposit({blockNumber, index}: {blockNumber: number; index: numbe
     depositData: {} as phase0.DepositData, // Not used
   };
 }
+
+function pick<T, K extends keyof T>(obj: T, keys: K[] | string[]): Pick<T, K> {
```

Comment:
Why do we need both input types? If we can't make it work with a generic `Pick`, I guess only `string[]` would do.

Reply:
Yeah, a bit complicated.